src/utils/utility.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { products } from './links';
+
+const getCategoryGroupByName = category => {
+  const categoryGroup = products.find(categoryGroup => categoryGroup.category.toLowerCase() === category.toLowerCase());
+  return categoryGroup;
+};
+
+const getCategoryGroupById = id => {
+  const categoryGroup = products.find(categoryGroup => categoryGroup.id.toString() === id.toString());
```

Comment:
`return products.find(categoryGroup => categoryGroup.id.toString() === id.toString())` could be a shorter way of doing this without requiring the assignment?

Reply:
OK, no problem. I'll change it.
If you ask me, that's my habit when I was working in C++, I just learned from one senior to me. :)